<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialEditor</class>
 <widget class="QDialog" name="MaterialEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Material Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="QtIcons.qrc">
    <normaloff>:/QtIcons/materialeditor.png</normaloff>:/QtIcons/materialeditor.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QToolBar" name="toolbar">
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <addaction name="actionNewMaterial"/>
     <addaction name="actionCloneMaterial"/>
     <addaction name="separator"/>
     <addaction name="actionShowAll"/>
     <addaction name="actionMaterialsInstances"/>
     <addaction name="actionInstances"/>
     <addaction name="separator"/>
     <addaction name="actionAutoExpand"/>
     <addaction name="actionSwitchQuality"/>
     <addaction name="actionMaterialReload"/>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>3</number>
     </property>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="MaterialTree" name="materialTree">
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragDrop</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="uniformRowHeights">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Template:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="templateBox">
            <property name="maxVisibleItems">
             <number>20</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QtPropertyEditor" name="materialProperty"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionNewMaterial">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="QtIcons.qrc">
     <normaloff>:/QtIcons/sphere_add.png</normaloff>:/QtIcons/sphere_add.png</iconset>
   </property>
   <property name="text">
    <string>New Material</string>
   </property>
  </action>
  <action name="actionCloneMaterial">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="QtIcons.qrc">
     <normaloff>:/QtIcons/sphere-clone.png</normaloff>:/QtIcons/sphere-clone.png</iconset>
   </property>
   <property name="text">
    <string>Clone Material</string>
   </property>
  </action>
  <action name="actionAutoExpand">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QtIcons.qrc">
     <normaloff>:/QtIcons/expand_all.png</normaloff>:/QtIcons/expand_all.png</iconset>
   </property>
   <property name="text">
    <string>autoExpand</string>
   </property>
   <property name="toolTip">
    <string>Turn on/off automatin node expanding</string>
   </property>
  </action>
  <action name="actionSwitchQuality">
   <property name="text">
    <string>switchQuality</string>
   </property>
  </action>
  <action name="actionMaterialReload">
   <property name="text">
    <string>MaterialReload</string>
   </property>
  </action>
  <actiongroup name="filterType">
   <action name="actionShowAll">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="icon">
     <iconset resource="QtIcons.qrc">
      <normaloff>:/QtIconsTextureDialog/opacity.png</normaloff>:/QtIconsTextureDialog/opacity.png</iconset>
    </property>
    <property name="text">
     <string>Show all</string>
    </property>
    <property name="toolTip">
     <string>Shows all materials and instances</string>
    </property>
   </action>
   <action name="actionMaterialsInstances">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="icon">
     <iconset resource="QtIcons.qrc">
      <normaloff>:/QtIconsTextureDialog/green.png</normaloff>:/QtIconsTextureDialog/green.png</iconset>
    </property>
    <property name="text">
     <string>Show Materials and Instances</string>
    </property>
    <property name="toolTip">
     <string>Show all Materials and only selected Instances</string>
    </property>
   </action>
   <action name="actionInstances">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="icon">
     <iconset resource="QtIcons.qrc">
      <normaloff>:/QtIconsTextureDialog/blue.png</normaloff>:/QtIconsTextureDialog/blue.png</iconset>
    </property>
    <property name="text">
     <string>Show Instances</string>
    </property>
    <property name="toolTip">
     <string>Show only selected Materials and Instances</string>
    </property>
   </action>
  </actiongroup>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtPropertyEditor</class>
   <extends>QTreeView</extends>
   <header>Tools/QtPropertyEditor/QtPropertyEditor.h</header>
  </customwidget>
  <customwidget>
   <class>MaterialTree</class>
   <extends>QTreeView</extends>
   <header>MaterialEditor/MaterialTree.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QtIcons.qrc"/>
 </resources>
 <connections/>
</ui>
